src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -92,14 +92,25 @@ internal class RazorSemanticTokensLegend
             new SemanticTokenType("extension method name"), // 30
             new SemanticTokenType("field name"),
             new SemanticTokenType("interface name"),
+            new SemanticTokenType("json - array"),
```

Comment:
Ya totally agree, this doesn't seem like a sustainable way to add more support for embedded embedded languages in C#. 

@CyrusNajmabadi how would you feel about generalizing C# embedded language classifications? Aka, "Embedded array, embedded comment etc."? Asking because if we continue to add this type of support then in VS, VSCode (basically any platform trying trying to consume these changes) there will be a requirement for the consumers of the C# side to chase down these classifications in order to populate their classifications properly.